pages/tutorials/getting_started.md.erb

```diff
@@ -42,8 +42,13 @@ Congratulations, you have run your first Buildkite build! :tada:
 
 To invite your team so they can see your build, go to your organization’s *Settings*, and under *Users* you’ll be able to paste in their email addresses.
 
-## Using a private repository
+## Use a private repository (optional)
 
 When you create a new pipeline with a private repository URL you’ll be shown instructions for configuring your SCM’s webhooks (in GitHub, Bitbucket, etc). Once you’ve followed those instructions make sure your [agent’s SSH keys](/docs/agent/v3/ssh-keys) are configured, and you’ll be good to run a build of your private pipeline.
 
 For more advanced pipelines it can be a good idea to use your development machine as the agent for your first few builds. That way all the dependencies are ready and you’ll sooner be able to share a link to a green build with the rest of your team.
+
+<section class="Docs__note">
+  <h3>Enable extra features by upgrading (optional)</h3>
+  <p>Buildkite Standard and Buildkite Enterprise plans have [extra features](/docs/pipelines/standard-and-enterprise-features) that might be useful for your Buildkite organization. You can enable these extra features by upgrading your payment plan.</p>
```

Comment:
If you put content inside an html div, it needs to be html, not markdown. We've got an open ticket about fixing that experience.

Reply:
```suggestion
  <p>Buildkite Standard and Buildkite Enterprise plans have [extra features](/docs/pipelines/standard-and-enterprise-features) that might be useful for your organization.</p>
```